autoload/gh/issues.vim

```diff
@@ -64,6 +64,7 @@ function! s:set_issue_list(resp) abort
         \ 'url': url .. issue.number,
         \ 'labels': len(issue.labels) > 0 ? printf('(%s)', join(map(copy(issue.labels), {_, label -> label.name}), ", ")) : '',
         \ 'comment': issue.comments > 0 ? printf("\uf41f %d", issue.comments) : '',
+        \ 'assignees': len(issue.assignees) > 0 ? printf('@%s', join(map(copy(issue.assignees), {_, assignee -> assignee.login}), ", ")) : 'no-assignees',
```

Comment:
I think assignees should display it after the user.
And if assignee is not exist, then display whitespace.

```              
#111 open @skanehira @yoshio15 Add file to gist                                                             (gists, help wanted)                       
#110 open @skanehira           Can edit in comment list buffer when there no any comments                   (bug, issue)                               
#109 open @skanehira @yoshio15 Re-run github actions workflow                                               (enhancement, github actions, help wanted) 
#108 open @skanehira           edit pull request body                                                       (enhancement, help wanted, pulls)            
```